Bottom: 4c39bbe9d9967e123bc70f80985ca589bb10cdd6
Top:    3814a10aaef10bf4f02605079b32b725686b21fd
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-25 10:36:08 -0500

Refresh of fixup-tmp-rework-ip-address-network

---
diff --git a/roles/docker-svn/defaults/main.yml b/roles/docker-svn/defaults/main.yml
index e3e85d1..93ba705 100644
--- a/roles/docker-svn/defaults/main.yml
+++ b/roles/docker-svn/defaults/main.yml
@@ -1,12 +1,11 @@
 ---
 
 # The hostname passed as an envirnment variable into the container
-svn_network:
-  ip_addr: '127.0.0.1'
-  hostname: svn.example.com
-  port_args: 
-    - '{{ svn_network.ip_addr }}:80:80'
-    - '{{ svn_network.ip_addr }}:443:443'
+svn_ip_addr: '127.0.0.1'
+svn_hostname: svn.example.com
+svn_port_args: 
+  - '80'
+  - '443'
 
 # The repositories that will be created/imported durring a data-volume restore
 svn_repos: 'example'
